stackit/internal/services/argus/scrapeconfig/resource.go

```diff
@@ -713,11 +1358,16 @@ func toCreatePayload(ctx context.Context, model *Model, saml2Model *saml2Model,
 		ScrapeInterval: conversion.StringValueToPointer(model.ScrapeInterval),
 		ScrapeTimeout:  conversion.StringValueToPointer(model.ScrapeTimeout),
 		// potentially lossy conversion, depending on the allowed range for sample_limit
-		SampleLimit: utils.Ptr(float64(model.SampleLimit.ValueInt64())),
-		Scheme:      conversion.StringValueToPointer(model.Scheme),
+		SampleLimit:     utils.Ptr(float64(model.SampleLimit.ValueInt64())),
+		Scheme:          conversion.StringValueToPointer(model.Scheme),
+		BearerToken:     conversion.StringValueToPointer(model.BearerToken),
+		HonorLabels:     conversion.BoolValueToPointer(model.HonorLabels),
+		HonorTimeStamps: conversion.BoolValueToPointer(model.HonorTimeStamps),
 	}
 	setDefaultsCreateScrapeConfig(&sc, model, saml2Model)
 
+	var diags diag.Diagnostics
```

Comment:
I don't think this is necessary, was there some place you need this to be declared?

Reply:
As I was using in a few places, I thought it is better if I declare it in this place. But now I changed them to declaration whereever needed with :=